examples/federated_learning/surface_defect_detection/README.md

```diff
@@ -15,14 +15,19 @@ Follow the [Sedna installation document](/docs/setup/install.md) to install Sedn
 ### Prepare Dataset
 
 Download [dataset](https://github.com/abin24/Magnetic-tile-defect-datasets.) and the [label file](/examples/federated_learning/surface_defect_detection/data/1.txt) to `/data` of edge1.  
+```
+mkdir -p /data
+cd /data
+git clone https://github.com/abin24/Magnetic-tile-defect-datasets..git
+curl -o 1.txt https://raw.githubusercontent.com/kubeedge/sedna/main/examples/federated_learning/surface_defect_detection/data/1.txt
```

Comment:
there is a mis-consistent problem here, you can do one of:
1. Update the content of label file. see https://github.com/kubeedge/sedna/issues/25#issuecomment-806435945
2. update the git-clone `git clone https://github.com/abin24/Magnetic-tile-defect-datasets..git Magnetic-tile-defect-datasets`

Reply:
Modified it